offchain-modules/src/packages/db/model.ts

```diff
@@ -117,3 +117,21 @@ export function EosLock2CkbMint(record: IEosLock): CkbMint {
     recipientLockscript: record.memo,
   });
 }
+
+export interface IBtcLock {
+  txid: string;
+  txHash: string;
+  amount: string;
+  rawTx: string;
+  recipientAddress: string;
```

Comment:
shoule be data, which is serde data of `recipientAddress `, `sudtExtraData` or more fields.